### ISSUE TITLE
For rendering the segmentation results

### ISSUE THREAD

**Issue by Lizhinwafu** [Jumpat/SegAnyGAussians#36]
python render.py -m <path to the pre-trained 3DGS model> --precomputed_mask <path to the segmentation results> --target scene --segment

I have saved multiple precomputed_masks, how can I segment these multiple targets at once?

**Comment by Jumpat** [comment]
I think the segmentation phase has been finished after you obtained the precomputed_masks. Do you want to render them together? If so, you should merge these precomputed_masks and use the merged mask for rendering.

**Comment by Jumpat** [comment]
Yeah, the computer vision community is always chasing an end to end method.

However, the key problem of directly segmenting 3D Gaussian is the lack of training data. In other words, there are lots of labeled point clouds but much less labeled 3D Gaussians.

Another fact is that this 2D to 3D paradigm is very suitable to 3D-GS, due to the existence of the differentiable rendering tech. It makes mapping 2D segments to 3D intuitive and reasonable.
